scripts/setup-stagenet.sh

```diff
@@ -7,17 +7,14 @@ echo "starting monerod..."
 sleep 5
 
 echo "starting monero-wallet-rpc on port 18083..."
-nohup ./monero-bin/monero-wallet-rpc --rpc-bind-port 18083 --disable-rpc-login --wallet-dir ./bob-test-keys --stagenet --trusted-daemon
+nohup ./monero-bin/monero-wallet-rpc --rpc-bind-port 18083 --disable-rpc-login --wallet-dir ./node-keys --stagenet --trusted-daemon &> monero-wallet-cli.log &
```

Comment:
Should we try the `--detach` and `--log-file` flags to `monero-wallet-rpc`?

Reply:
that would be nice, didn't realize it had a `--detach` flag as well. I can update all of the scripts in a follow-up